flang/test/Driver/xlinker.f90

```diff
@@ -0,0 +1,6 @@
+! RUN: %flang -### -o /dev/null %s -Xlinker -rpath=/not/a/real/path 2>&1 | FileCheck %s
+
+! CHECK: "-fc1"
+! CHECK-NEXT: "-rpath=/not/a/real/path"
```

Comment:
nit: maybe I'm being overly pedantic here, but I think the meaning of the test would be clearer if you check directly that this is an argument to the linker, not assuming that the next line after the frontend driver is the linker. There are some regular expressions for names of the linker in `flang/test/Driver/linker-flags.f90`.

Reply:
Thanks. I have updated the tests quite extensively. There are now checks for various platforms and multiple occurrences of `-Xlinker`. How does this look?